src/vmp_stack.c

```diff
@@ -566,7 +569,15 @@ int vmp_native_enable(void) {
         if ((unw_is_signal_frame = dlsym(libhandle, UL_PREFIX PREFIX "_is_signal_frame")) == NULL) {
             goto bail_out;
         }
-        if ((unw_getcontext = dlsym(libhandle, U_PREFIX PREFIX "_getcontext")) == NULL) {
+#if __powerpc64__
+//getcontext() naming follows a different pattern on PPC64
+#define U_PREFIX
+#define PREFIX
+#define USCORE
+#else
+#define USCORE _
```

Comment:
```suggestion
#define USCORE
```

Reply:
Woops, I meant `"_"`

Reply:
Fixed in https://github.com/vmprof/vmprof-python/pull/239/commits/7f009cd3373158582ab308479c1eb60e9ce126b0